diff --git a/libavfilter/avfilter.c b/libavfilter/avfilter.c
index 831871de90..17dc424dc2 100644
--- a/libavfilter/avfilter.c
+++ b/libavfilter/avfilter.c
@@ -1208,16 +1208,16 @@ static int ff_filter_activate_default(AVFilterContext *filter)
 {
     unsigned i;
 
-    for (i = 0; i < filter->nb_outputs; i++) {
-        FilterLinkInternal *li = ff_link_internal(filter->outputs[i]);
-        int ret = li->status_in;
-
-        if (ret) {
-            for (int j = 0; j < filter->nb_inputs; j++)
-                ff_inlink_set_status(filter->inputs[j], ret);
-            return 0;
-        }
-    }
+    //for (i = 0; i < filter->nb_outputs; i++) {
+    //    FilterLinkInternal *li = ff_link_internal(filter->outputs[i]);
+    //    int ret = li->status_in;
+
+    //    if (ret) {
+    //        for (int j = 0; j < filter->nb_inputs; j++)
+    //            ff_inlink_set_status(filter->inputs[j], ret);
+    //        return 0;
+    //    }
+    //}
 
     for (i = 0; i < filter->nb_inputs; i++) {
         if (samples_ready(ff_link_internal(filter->inputs[i]),
